index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>CV of Aleksei Bulgak</title>
+  <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css" />
+</head>
+
+<body>
+  <header>
+    <div class="header container">
+      <span>Aleksei Bulgak</span>
+      <div class="social-networks">
+        <a href="https://github.com/aleksei-bulgak-study" target="_blank">
+          <img src="img/github.png" alt="Github" width="30" height="30" />
```

Comment:
better to use icon font

Reply:
if you use img inside a, you should make <a> an inline-block element and make font-size: 0 to avoid unnecessary extra space and other misunderstandings with size of elements